web-console/knikubevirt/vm-details/devices-and-resources/devices-and-respurces.md

```diff
@@ -0,0 +1,40 @@
+# VM details - Devices-and-resources
+
+### Virtual machine
+
+## From VM details page
+![VM details - Overview tab - default](img/VM-2-0.jpg)
+Within the VM details > overview tab > overview section, a click on the ‘Devices & resources’ field will pop up the correlated modal.
+
+![Devices & resources modal - default](img/VM-3-0.png)
+The ‘Devices & resources’ modal currently contains only the ‘workload isolation’ option. This may expand in the future. The modal is titled this way, instead of  ‘Workload isolation’, in order to align with the implementation on the pod’s side.
+
+A sub-text below the title mentions that the scheduler will attempt to achieve the following settings. In case it is not possible, the VM will be scheduled in the best way it can (the same thing applies for pods, where this modal has more settings).
+
+The user selects ‘Isolate this workload…’ checkbox.
+
+![VM details - Overview tab - workload isolation enabled](img/VM-4-0.jpg)
+The preview on the ‘Devices & resources’ us updated to reflect the changes.
+
+## From VM creation wizard
+![Create VM - Advanced - Virtual hardware](img/Create-VM-4-2-00.jpg)
```

Comment:
This should be CD-ROMs instead of Guest agent. See here
https://codepen.io/matthewcarleton/pen/OJJazap?editors=1100.

Reply:
same comment as above about the Devices being dropped and maybe titling this something more in line with the flavor concept.

Reply:
I need to sync with you on this new iteration of Virtual hardware. Seems that some stuff had changed like dropping the 'Mount this guest agent installer' checkbox, which probably should be a radio button now. Dropping the support text. Maybe I should leave that 'CD-ROM section blurred out?